addons/notes/README.md

```diff
@@ -16,6 +15,7 @@ Storybook Addon Notes allows you to write notes (text or HTML) for your stories
 ![Storybook Addon Notes Demo](docs/demo.png)
 
 ### Getting Started
+**NOTE: Documentation on master branch is for alpha version, stable release is on [release/3.4](https://github.com/storybooks/storybook/tree/release/3.4/addons/)**
```

Comment:
It would probably make sense to specify the particular addon in each link

Reply:
Indeed, very very valuable 🙇